en/dev_log/logging.md

```diff
@@ -100,33 +100,41 @@ amount of dropouts:
 
-| SD Card                                                       | Mean Seq. Write Speed [KB/s] | Max Write Time / Block (average) [ms] |
-| ------------------------------------------------------------- | ---------------------------- | ------------------------------------- |
-| SanDisk Extreme U3 32GB                                       | 461                          | **15**                                |
-| Sandisk Ultra Class 10 8GB                                    | 348                          | 40                                    |
-| Sandisk Class 4 8GB                                           | 212                          | 60                                    |
-| SanDisk Class 10 32 GB (High Endurance Video Monitoring Card) | 331                          | 220                                   |
-| Lexar U1 (Class 10), 16GB High-Performance                    | 209                          | 150                                   |
-| Sandisk Ultra PLUS Class 10 16GB                              | 196                          | 500                                   |
-| Sandisk Pixtor Class 10 16GB                                  | 334                          | 250                                   |
-| Sandisk Extreme PLUS Class 10 32GB                            | 332                          | 150                                   |
-
-More important than the mean write speed is the maximum write time per block (of 4 KB).
-This defines the minimum buffer size: the larger this maximum, the larger the log buffer needs to be to avoid dropouts.
-Logging bandwidth with the default topics is around 50 KB/s, which all of the SD cards satisfy.
+The table below shows the **mean sequential write speed [KB/s]** / **maximum write time per block (average) [ms]** for F4- (Pixracer), F7-, and H7-based flight controllers.
+
+| SD Card                                                       | F4           | F7  | H7  |
+| ------------------------------------------------------------- | ------------ | --- | --- |
+| SanDisk Extreme U3 32GB                                       | 461 / **15** | ?/? | ?/? |
+| Samsung EVO Plus 32                                           | ?/?          | ?/? | ?/? |
+| Sandisk Ultra Class 10 8GB                                    | 348 / 40     | ?/? | ?/? |
+| Sandisk Class 4 8GB                                           | 212 / 60     | ?/? | ?/? |
+| SanDisk Class 10 32 GB (High Endurance Video Monitoring Card) | 331 / 220    | ?/? | ?/? |
+| Lexar U1 (Class 10), 16GB High-Performance                    | 209 / 150    | ?/? | ?/? |
+| Sandisk Ultra PLUS Class 10 16GB                              | 196 /500     | ?/? | ?/? |
+| Sandisk Pixtor Class 10 16GB                                  | 334 / 250    | ?/? | ?/? |
+| Sandisk Extreme PLUS Class 10 32GB                            | 332 / 150    | ?/? | ?/? |
```

Comment:
Can I please get some help filling out this table for F7, H7 flight controllers, and also for the Samsung EVO plus 32 on F4?

First number is the mean sequential write speed, second is the average write time per block.

Reply:
```
  SanDisk Extreme U3 32GB
  - F4: 1500kb/s / 15 ms
  - F7: 1800kb/s / 10 ms
  - H7: 2900kb/s / 8 ms
  Samsung EVO Plus 32GB
  - F4: 1700kb/s / 10-60 ms
  - F7: 1800kb/s / 10-60 ms
  - H7: 1900kb/s / 9-60 ms
```

Reply:
These are the spikes. We can also just note the higher value.

Reply:
Thanks, Happy to keep as is.